CodeComplianceTest_Engine/Query/Checks/HasPublicGet.cs

```diff
@@ -42,7 +42,10 @@ public static partial class Query
         public static Span HasPublicGet(PropertyDeclarationSyntax node)
         {
             AccessorDeclarationSyntax getter = node.PropertyGetAccessor();
-            return (getter == null || getter.IsPrivate()) ? getter.Modifiers.Span.ToBHoM() : null;
+            if (getter == null)
+                return node.Modifiers.Span.ToBHoM();
```

Comment:
Could we use maybe the span of the name or something rather than of the property modifiers?

Reply:
According to intellisense `PropertyDeclarationSyntax` did not have a name property, but it did have its own span so I've changed to that if that's ok?